<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="5" passed="4" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-06-20T02:11:48 IST" name="Suite" finished-at="2024-06-20T02:12:55 IST" duration-ms="67177">
    <groups>
    </groups>
    <test started-at="2024-06-20T02:11:48 IST" name="Testing" finished-at="2024-06-20T02:12:55 IST" duration-ms="67177">
      <class name="Com.TestCases.TC_05_Add_one_item_and_place_order">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:Com.TestCases.TC_05_Add_one_item_and_place_order@f58853c]" started-at="2024-06-20T02:12:39 IST" name="launchApplication" finished-at="2024-06-20T02:12:46 IST" duration-ms="7221" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="verify_TC_04_Delete_item_from_cart_and_verify_item_removed_successfully()[pri:0, instance:Com.TestCases.TC_05_Add_one_item_and_place_order@f58853c]" started-at="2024-06-20T02:12:46 IST" name="verify_TC_04_Delete_item_from_cart_and_verify_item_removed_successfully" finished-at="2024-06-20T02:12:53 IST" duration-ms="6821" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verify_TC_04_Delete_item_from_cart_and_verify_item_removed_successfully -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:Com.TestCases.TC_05_Add_one_item_and_place_order@f58853c]" started-at="2024-06-20T02:12:53 IST" name="tearDown" finished-at="2024-06-20T02:12:55 IST" duration-ms="1523" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- Com.TestCases.TC_05_Add_one_item_and_place_order -->
      <class name="Com.TestCases.TC_03_Verify_home_page_demo_tesst_case">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:Com.TestCases.TC_03_Verify_home_page_demo_tesst_case@6e2aa843]" started-at="2024-06-20T02:12:22 IST" name="launchApplication" finished-at="2024-06-20T02:12:27 IST" duration-ms="5220" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="verify_TC_03_Verify_methods()[pri:0, instance:Com.TestCases.TC_03_Verify_home_page_demo_tesst_case@6e2aa843]" started-at="2024-06-20T02:12:27 IST" name="verify_TC_03_Verify_methods" finished-at="2024-06-20T02:12:28 IST" duration-ms="652" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [4] but found [3]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [4] but found [3]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1419)
at org.testng.Assert.assertEquals(Assert.java:1383)
at org.testng.Assert.assertEquals(Assert.java:1429)
at Com.TestCases.TC_03_Verify_home_page_demo_tesst_case.verify_TC_03_Verify_methods(TC_03_Verify_home_page_demo_tesst_case.java:32)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verify_TC_03_Verify_methods -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:Com.TestCases.TC_03_Verify_home_page_demo_tesst_case@6e2aa843]" started-at="2024-06-20T02:12:28 IST" name="tearDown" finished-at="2024-06-20T02:12:30 IST" duration-ms="1274" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- Com.TestCases.TC_03_Verify_home_page_demo_tesst_case -->
      <class name="Com.TestCases.TC_01_Login_to_application">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:Com.TestCases.TC_01_Login_to_application@7486b455]" started-at="2024-06-20T02:11:48 IST" name="launchApplication" finished-at="2024-06-20T02:12:07 IST" duration-ms="19358" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="verify_TC_01_Login_to_application()[pri:0, instance:Com.TestCases.TC_01_Login_to_application@7486b455]" started-at="2024-06-20T02:12:07 IST" name="verify_TC_01_Login_to_application" finished-at="2024-06-20T02:12:09 IST" duration-ms="1678" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verify_TC_01_Login_to_application -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:Com.TestCases.TC_01_Login_to_application@7486b455]" started-at="2024-06-20T02:12:09 IST" name="tearDown" finished-at="2024-06-20T02:12:11 IST" duration-ms="1411" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- Com.TestCases.TC_01_Login_to_application -->
      <class name="Com.TestCases.TC_04_Delete_item_from_cart_and_verify_item_removed_successfully">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:Com.TestCases.TC_04_Delete_item_from_cart_and_verify_item_removed_successfully@6f36c2f0]" started-at="2024-06-20T02:12:30 IST" name="launchApplication" finished-at="2024-06-20T02:12:36 IST" duration-ms="5942" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="verify_TC_04_Delete_item_from_cart_and_verify_item_removed_successfully()[pri:0, instance:Com.TestCases.TC_04_Delete_item_from_cart_and_verify_item_removed_successfully@6f36c2f0]" started-at="2024-06-20T02:12:36 IST" name="verify_TC_04_Delete_item_from_cart_and_verify_item_removed_successfully" finished-at="2024-06-20T02:12:38 IST" duration-ms="1948" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verify_TC_04_Delete_item_from_cart_and_verify_item_removed_successfully -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:Com.TestCases.TC_04_Delete_item_from_cart_and_verify_item_removed_successfully@6f36c2f0]" started-at="2024-06-20T02:12:38 IST" name="tearDown" finished-at="2024-06-20T02:12:39 IST" duration-ms="1406" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- Com.TestCases.TC_04_Delete_item_from_cart_and_verify_item_removed_successfully -->
      <class name="Com.TestCases.TC_02_Add_items_to_cart_and_verify">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:Com.TestCases.TC_02_Add_items_to_cart_and_verify@69ee81fc]" started-at="2024-06-20T02:12:11 IST" name="launchApplication" finished-at="2024-06-20T02:12:19 IST" duration-ms="8544" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="verify_TC_02_Add_items_to_cart_and_verify()[pri:0, instance:Com.TestCases.TC_02_Add_items_to_cart_and_verify@69ee81fc]" started-at="2024-06-20T02:12:19 IST" name="verify_TC_02_Add_items_to_cart_and_verify" finished-at="2024-06-20T02:12:21 IST" duration-ms="1777" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verify_TC_02_Add_items_to_cart_and_verify -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:Com.TestCases.TC_02_Add_items_to_cart_and_verify@69ee81fc]" started-at="2024-06-20T02:12:21 IST" name="tearDown" finished-at="2024-06-20T02:12:22 IST" duration-ms="1331" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- Com.TestCases.TC_02_Add_items_to_cart_and_verify -->
    </test> <!-- Testing -->
  </suite> <!-- Suite -->
</testng-results>
